py4DSTEM/process/fit/fit.py

```diff
@@ -107,7 +107,7 @@ def fit_2D(
                 fit_mean_square_error
                 > np.mean(fit_mean_square_error) * robust_thresh**2
             )
-            mask[_mask] == False
+            mask[_mask] is False
```

Comment:
I'm not sure what the intended behavior of the original code was here. It looks like this should have been an assignment.

Reply:
changing to

Reply:
It might blame to @smribet because of the big reformatting PR. I suspect @cophus or @bsavitzky wrote the code here

Reply:
Pretty sure this was me.  Why are we changing every boolean `==` to `is` now?  Does this PR mean I have to read PEP-8?  I feel like this is not universally desirable behavior...

Reply:
Ok I reviewed and this (a) was Colin and (b) def should be an assignment :p

Reply:
changed to assignment